pyproject.toml

```diff
@@ -18,6 +18,7 @@ select = [
     "RUF",     # ruff rules
     "T10",     # flake8-debugger
     "TCH",     # flake8-type-checking
+    "UP032",   # f-string
```

Comment:
Cool, we should have this everywhere :)

Reply:
https://github.com/cockpit-project/starter-kit/pull/708